webui/react/src/components/Grid.tsx

```diff
@@ -12,6 +12,7 @@ export enum GridMode {
 interface Props {
   border?: boolean;
   gap?: ShirtSize;
+  length?: number;
```

Comment:
Looking to how this is used, couldn't just update `mode?: GridMode|number`?

Reply:
yep, that's probably what a smart person would've done! :D